elixir/apps/api/lib/api/relay/channel.ex

```diff
@@ -23,6 +23,14 @@ defmodule API.Relay.Channel do
     end
   end
 
+  def handle_info("disconnect", socket) do
```

Comment:
@thomaseizinger @conectado now relays can receive `disconnect` message too in case their token is deleted. But generally we will close the TCP socket whenever that happens right away (it's not always that Phoenix allows us to broadcast it before it happens).

Reply:
Okay. I think at the moment, we don't expect any inbound messages on the relay so we will just log that as a deserialization error and continue. Will the socket always disconnect?

Reply:
Yes, we will always close the TCP socket when it happens. I think for now we just need to prevent crashing on message deserialization.

Reply:
We don't crash on unknown messages, just logging a warning!